spec/services/moves/updater_spec.rb

```diff
@@ -6,8 +6,9 @@
   subject(:updater) { described_class.new(move, move_params) }
 
   let(:before_documents) { create_list(:document, 2) }
-  let!(:from_location) { create :location }
-  let!(:move) { create :move, :proposed, move_type: 'prison_recall', from_location: from_location, documents: before_documents }
+  let!(:from_location) { create(:location) }
+  let!(:move) { create(:move, :proposed, move_type: 'prison_recall', from_location: from_location, profile: profile) }
```

Comment:
Maybe in the meantime we could remove the bang method?
a solution could be put them in a `before do`

Reply:
All of this code is going away so I'm not keen to refactor it.